pkgs/desktops/gnome/core/gnome-keyring/default.nix

```diff
@@ -51,8 +51,7 @@ stdenv.mkDerivation rec {
     p11-kit
   ];
 
-  # In 3.20.1, tests do not support Python 3
-  checkInputs = [ dbus python2 ];
+  checkInputs = [ dbus python3 ];
```

Comment:
Tests are disabled so it should not even be brought into build closure.

Can you check that the tests support Python 3 now (even if they were to fail)?

Reply:
Will check this afternoon, missed the `doCheck = false` Dx

Reply:
What I do remember though is that I changed `python2 = python27` to `python2 = throw "gotcha"` in a local copy yesterday to find everything that did use python2 in my NixOS configuration and this did come up.

Reply:
It might still be passed as `checkInputs` environment variable (unless explicitly filtered out), just not appended to `nativeBuildInputs`.